taskcluster/README.md

```diff
@@ -0,0 +1,3 @@
+# taskcluster
+
+A task execution framework for Mozilla's CI system. Please consult the [existing taskcluster documentation](https://docs.taskcluster.net/docs).
```

Comment:
Could you make it clear that these are files for taskcluster not taskcluster itself.

Reply:
commit 54d4c6fa4c0b6d5e21cc42bcfc3d7ba114197dd8